nicolegooden/indecision-maker
# Refactor design and display

#### What's this PR do?  

+ Corrects some misspelled words
+ Makes phrases and descriptors for activity details more clear
+ Removes scroll bars from result image
+ Updates README with proper installation instructions
+ Adjusts styling, specifically for the result component
+ Changes date object for release date of activities on result page to human-readable dates
+ Fixes bug related to the `similar picks` from the result page
  + only the last selected activity's suggestions were showing up
  + now, all suggestions from all activities exist as similar picks

#### Where should the reviewer start? 

Most of the changes occur in `ResultPage.js` and `ResultPage.scss`.

#### How should this be manually tested?  

Run this locally to ensure all functionality is working as expected, mostly for the form and result views.

#### Any background context you want to provide?  

I believe these small changes make for a better user experience, especially when viewing the suggested activity and similar picks via the `ResultPage` component.  Please let me know if there is anything to improve upon before merging!  Thanks.

#### What are the relevant tickets?  

Closes #73 
Closes #74 
Closes #75 
Closes #76 
Closes #77 
Closes #78
